modules/types/tests/tests.cpp

```diff
@@ -37,17 +38,16 @@ TYPED_TEST(TypeTests, RandomUnequalTest) {
 // Test types which support hashing
 //=================================================================================================
 template <class T>
-class HashingTest : public ::testing::Test {};
+class HashingTest : public heph::test_utils::HephTest {};
+
 using HashingImplementations = ::testing::Types<UuidV4>;
 
 TYPED_TEST_SUITE(HashingTest, HashingImplementations);
 
 TYPED_TEST(HashingTest, TestHash) {
-  auto mt = random::createRNG();
-
   EXPECT_TRUE(absl::VerifyTypeImplementsAbslHashCorrectly({
       TypeParam(),
-      TypeParam::random(mt),
+      TypeParam::random(this->mt),
```

Comment:
`this->` is necessary in templated tests to find the base class member.

Reply:
shuoldn't we maybe make `mt` private and add `mt()`? To make it more explicit that it comes from the inheritance?